### PR TITLE
Add UAE dummydata

### PR DESCRIPTION
**Pull request by hazemakhalek**
## Changes proposed in this Pull Request
This PR adds the necessary dummy data for the workflow to run for UAE

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [x] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.

### CODE REVIEW

File: scripts/prepare_sector_network.py [hunk 1]
```diff
@@ -752,6 +753,10 @@ def add_aviation(n, cost):
     )
 
 
+def new_func():
```
Comment (energyLS) [inline scripts/prepare_sector_network.py:756]: What is the purpose of this function?

File: scripts/prepare_sector_network.py [hunk 2]
```diff
@@ -2161,8 +2166,8 @@ def add_rail_transport(n, costs):
     n = pypsa.Network(snakemake.input.network, override_component_attrs=overrides)
 
     # Fetch the coutry list from the network
-    countries = list(n.buses.country.unique())
-
+    # countries = list(n.buses.country.unique())
+    countries = snakemake.config["countries"]
```
Comment (energyLS) [inline scripts/prepare_sector_network.py:2170]: What is the advantage fetching the countries from the config? Isn't it more failsafe to get it from the network?

**Review by energyLS — APPROVED** [review]

File: scripts/prepare_sector_network.py (hunk 2 above)
Comment (Hazem-IEG) [reply scripts/prepare_sector_network.py:2170]: Yes and it also avoids breaking the workflow in cases like this: https://github.com/pypsa-meets-earth/pypsa-earth/issues/649

File: scripts/prepare_sector_network.py (hunk 1 above)
Comment (Hazem-IEG) [reply scripts/prepare_sector_network.py:756]: As you can see, nothing at all 😆 
My bad. I probably pressed something in vscode that created it automatically.

**Pull request merged**